src/prompt/tooltip.go

```diff
@@ -56,6 +43,15 @@ func (e *Engine) Tooltip(tip string) string {
 
 	switch e.Env.Shell() {
 	case shell.PWSH, shell.PWSH5:
+		defer func() {
+			if e.fetchPromptCache() {
```

Comment:
I would rename that `CachePrompt()` as this isn't about fetching.

Reply:
Not sure if I get your point. Do you mean this is not about "fetching cache from file"? Here it tries to check any existing prompt cache and fill corresponding fields. The return value indicates whether a prompt cache is available.

Reply:
Renamed to `checkPromptCache` now.